graph-builder/src/config/file.rs

```diff
@@ -143,7 +143,7 @@ mod tests {
 
                 [upstream]
                 method = "registry"
-                pause_secs = 35
+                pause_secs = 600
```

Comment:
There's no point in keeping this in sync with the default, it's just a parser test.

Reply:
Dropped with f7c476f -> 000eacc.